src/tests/test_dependency.py

```
@@ -0,0 +1,44 @@
+import unittest
+from revolve.functions import *
```

Ruff (0.11.9)

`from revolve.functions import *` used; unable to detect undefined names

(F403)